apps/admin-gui/src/app/vos/components/notification-list/notification-list.component.ts

```diff
@@ -99,12 +108,14 @@ export class NotificationListComponent implements OnChanges, AfterViewInit {
         .setSendingEnabled({ mails: [applicationMail], enabled: false })
         .subscribe(() => {
           applicationMail.send = false;
+          this.notificator.showSuccess(this.disabledSendingMessage);
```

Comment:
Preferably use the `showInstantSuccess` method of the notificator. It handles the translation itself.